release-scripts/mandrel-release.java

```diff
@@ -479,6 +488,158 @@ private MandrelVersion getCurrentVersion()
+        };
+        final URI releaseEndpoint = URI.create("https://vendors.sdkman.io/release");
+        final URI makeDefaultEndpoint = URI.create("https://vendors.sdkman.io/default");
+        final URI announceEndpoint = URI.create("https://vendors.sdkman.io/announce/struct");
+        final HttpClient hc = HttpClient.newHttpClient();
+        for (int i = 0; i + 3 <= assets.length; i = i + 3)
+        {
+            if (assets[i].exists() && assets[i + 1].exists() && assets[i + 2].exists())
+            {
+                final Matcher m = filePattern.matcher(assets[i].getName());
+                if (m.matches())
+                {
+                    final String os = m.group("os").toUpperCase();
+                    final String arch = "amd64".equals(m.group("arch")) ? "64" : "ARM64";
+                    final String version = m.group("version") + ".r" + m.group("javaVersion").trim();
+                    versionsToPromote.add(version);
+                    final String sha1 = Files.readString(assets[i + 1].toPath(), StandardCharsets.UTF_8).split(" ")[0].trim().toLowerCase();
+                    final String sha256 = Files.readString(assets[i + 2].toPath(), StandardCharsets.UTF_8).split(" ")[0].trim().toLowerCase();
+                    final String url = "https://github.com/graalvm/mandrel/releases/download/mandrel-" + m.group("version") + "-" + suffix + "/" + assets[i].getName();
+                    info("Releasing: " + assets[i].getName());
+                    // Release
+                    final String releasePayload = "" +
+                        "{\n" +
+                        "    \"candidate\": \"java\",\n" +
+                        "    \"version\": \"" + version + "\",\n" +
+                        "    \"vendor\": \"mandrel\",\n" +
+                        "    \"url\": \"" + url + "\",\n" +
+                        "    \"platform\": \"" + os + "_" + arch + "\",\n" +
+                        "    \"checksums\": {\n" +
+                        "        \"SHA-1\": \"" + sha1 + "\",\n" +
+                        "        \"SHA-256\": \"" + sha256 + "\"\n" +
+                        "    }\n" +
+                        "}";
+                    final HttpRequest releaseRequest = HttpRequest.newBuilder()
+                        .POST(HttpRequest.BodyPublishers.ofString(releasePayload))
+                        .uri(releaseEndpoint)
+                        .headers(headers)
+                        .build();
+                    final HttpResponse<String> releaseResponse = hc.send(releaseRequest, HttpResponse.BodyHandlers.ofString());
+                    info("Released Response Code : " + releaseResponse.statusCode());
+                    info("Released Response Body : " + releaseResponse.body());
+                }
+                else
+                {
+                    error("File name " + assets[i].getName() + " does not match regexp " + filePattern.pattern() + " and that is unexpected. Skipping asset.");
```

Comment:
```suggestion
                    warn("File name " + assets[i].getName() + " does not match regexp " + filePattern.pattern() + " and that is unexpected. Skipping asset.");
```

or

```suggestion
                    error("File name " + assets[i].getName() + " does not match regexp " + filePattern.pattern() + " and that is unexpected. Aborting.");
```
error is aborting the whole process it's not just skipping the asset.